crates/dev-maker/fetch_rates.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
+# A script to fetch historic rates from Bitmex between a specifc time window
+
+# BitMEX API base URL
+API_URL="https://www.bitmex.com/api/v1"
+
+SYMBOL="XBTUSD"
+
+# Set start and end timestamps for the date range (March 20, 2023, to March 20, 2024)
+START_TIMESTAMP="2023-03-20T00:00:00.000Z"
+END_TIMESTAMP="2024-03-20T00:00:00.000Z"
+
+# Construct the request URL for hourly data
+REQUEST_URL="${API_URL}/trade/bucketed?symbol=${SYMBOL}&binSize=1h&partial=false&count=1000&reverse=false&startTime=${START_TIMESTAMP}&endTime=${END_TIMESTAMP}"
```

Comment:
nit: We only fetch the first 1000 prices here, which gets us to April 2023. We might want to have more recent prices?

Reply:
feel free to update it, I think for local testing it doesn't matter much.